mongorestore/options.go

```diff
@@ -16,6 +25,27 @@ or use -d and -c to restore a single collection from a single .bson file.
 
 See http://docs.mongodb.org/manual/reference/program/mongorestore/ for more information.`
 
+// Options defines the set of all options for configuring mongorestore.
+type Options struct {
+	*options.ToolOptions
+	*InputOptions
+	*NSOptions
+	*OutputOptions
+	TargetDirectory string
+}
+
+// InputOptions command line argument long names
+const (
```

Comment:
Because we're using these in the tests, I added constants for all the command line arguments. These will need to be kept up to date with the labels in the various options types. If they're tested properly (which not all of them are), tests will break when they become out of date.

Reply:
ack